examples/cosmos-etcd/readme.md

```diff
@@ -1,9 +1,63 @@
 # Kubernetes on Cosmos
 
-This example deployment allows you to bootstrap a kubernetes cluster on Azure that uses Cosmos instead of locally installed and configured etcd.
+The Azure Kubernetes Engine (aks-engine) generates [ARM](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-overview) (Azure Resource Manager) templates for Kubernetes clusters on Microsoft Azure. The input to aks-engine is a cluster definition file which describes the desired cluster, including orchestrator, features, and agents. 
 
-## Preview Instruction
-Please follow instruction in https://aka.ms/cosmosetcdapi-signup to enroll your subscription in the private preview preview of Cosmos etcd api.
+This example deployment allows you to use aks-engine to bootstrap a Kubernetes cluster on Azure that uses [Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/) instead of locally installed and configured etcd.
 
-## Get Support
-To get support during the preview you can file github issues on this repo. Please mention cosmosetcd team when filing the issue. 
+## What is the Azure Cosmos DB etcd API?
+[Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/) is a globally distributed, multi-model database service. It offers turnkey global distribution, guarantees single-digit millisecond latencies at the 99th percentile, elastic scaling of throughput and storage, and comprehensive SLA’s.
+
+The Azure Cosmos DB etcd API allows you to use Azure Cosmos DB as the backing store for Kubernetes. Azure Cosmos DB implements the wire level protocol of etcd, allowing master nodes’ API servers to communicate with and store data Azure Cosmos DB as if it were locally installed etcd. 
+
+## Prerequisites 
+1.	Install the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
+2.	Install [AKS-Engine](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/quickstart.md)
+3.	Have a whitelisted subscription for the Azure Cosmos DB etcd API. Follow instructions at https://aka.ms/cosmosetcdapi-signup to enroll in the private preview. 
+
+## Steps
+Follow the instructions [here](https://github.com/Azure/aks-engine/blob/master/docs/tutorials/quickstart.md#deploy-your-first-cluster) to deploy your first Kubernetes cluster. 
+
+Follow the instructions here to deploy your first Kubernetes cluster. 
```

Comment:
```suggestion
Follow the instructions here to deploy your first Kubernetes cluster backed by cosmos etcd. 
```

Reply:
The linked instructions don't show how to use the Azure Cosmos DB etcd API, which is described in this doc. Left the wording as is.